### ISSUE TITLE
SMP repository?

### ISSUE THREAD

**Issue by gordonfreemanhr** [jonaski/strawberry#240]
This is not really an issue, but a question: how do I add the Strawberry repository on Ubuntu 19.04?

Thanks!

**Comment by jonaski** [comment]
Not sure what repository you referer to. Maybe ask Ubuntu if it's still not in their repository?

**Comment by gordonfreemanhr** [comment]
You’re probably right. Let me rephrase: I’m a Ubuntu user. I’d like to keep my SMP up to date. Do you have any advice for me?

**Comment by jonaski** [comment]
I don't have a repository for ubuntu so I suggest to contact ubuntu (or debian) and ask about the progress of the packaging. See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=913079
Otherwise you need to manually download the deb file each time there is a release or use the snap, but I don't really recommend using snap.
